docs/benchmarks/data-comparisson.md

```diff
@@ -628,13 +628,16 @@ documents, and have the following meanings:
         c. BYPASS indicates that (in the original data) the story was
 		marked with the string "bypass" (or a typographical variant on that
 		string).
-		Altough this shouldn't be used for Topic search because there could be topics if there is a no even so no if theres a yes
+		Although this shouldn't be used for Topic search because there could be topics even if there is a NO, or likewise, there could be be no topics even if there is a YES.
```

Comment:
I copied this and the followeing 4 changes from their side, I wouldn't change everything they wrote

Reply:
> I copied this and the followeing 4 changes from their side, I wouldn't change everything they wrote

Humm... as a first time reader that was one of the lines that I had the biggest issue with. Thoses "if there...no even so no if" are very contrived and hard to understand. There's got to be a better way the phrase this. That was my suggestion, but even now I'm not even close to 100% sure on it.